src/world/Actor/Player.cpp

```diff
@@ -503,8 +511,18 @@ bool Sapphire::Entity::Player::exitInstance()
 {
   auto& teriMgr = Common::Service< TerritoryMgr >::ref();
 
-  auto pZone = getCurrentTerritory();
-  auto pInstance = pZone->getAsInstanceContent();
+  if( auto d = getCurrentTerritory()->getAsDirector() )
+    if( d->getContentFinderConditionId() > 0 )
```

Comment:
dont put the assignment in the conditional, do it outside and then `d && d->blah`